server/config.go

```diff
@@ -152,3 +154,23 @@ func (c *Config) ToFile(path string) error {
 
 	return ioutil.WriteFile(path, w.Bytes(), 0644)
 }
+
+// Check resolves that all paths and files are available as expected
+func (c *Config) Check() error {
+
+	for _, file := range []string{
+		c.RootConfig.Root,
+		c.RootConfig.SandboxDir,
+		c.RootConfig.ContainerDir,
+		c.RuntimeConfig.Runtime,
+		c.RuntimeConfig.Conmon,
+		c.RuntimeConfig.SeccompProfile,
+		c.ImageConfig.Pause,
+	} {
+		if _, err := os.Stat(file); err != nil && os.IsNotExist(err) {
+			return fmt.Errorf("%q does not exist", file)
```

Comment:
tests are failing in Travis also https://travis-ci.org/kubernetes-incubator/cri-o/jobs/183374758#L2137